src/gui/guiiconprovider.cpp

```diff
@@ -89,8 +89,7 @@ void GuiIconProvider::configure()
 #if (defined(Q_OS_UNIX) && !defined(Q_OS_MAC))
     m_useSystemTheme = Preferences::instance()->useSystemIconTheme();
     if (!m_useSystemTheme)
-        QIcon::setThemeName("qbt-theme");
-#else
-    QIcon::setThemeName("qbt-theme");
 #endif
+        QIcon::setThemeName("qbt-theme");
```

Comment:
@zeule we might have forgotten a use case here. What if user changes useSystemIconTheme() during runtime. Going from false to true in the same session will not have any effect.

Reply:
https://github.com/qbittorrent/qBittorrent/pull/9295#discussion_r208461792